FreeRTOS/FreeRTOS-Kernel
# Fix MISRA_C_2012 rule 17.7 violation

Fix MISRA C 2012 rule 17.7 violation

Description
-----------
**Misra Rule 17.7**

> The value returned by a function having non-void return type shall be used.

**MISRA violation**
The return type for the function xPortStartScheduler() is not void, as a result, it should be assigned to a non-void value 
The function prototype:
```
BaseType_t xPortStartScheduler()
{
 // code.
}
```

**Fix**

- Call the API with a void return type , since the return value is not getting used.

Test Steps
-----------
Run the Coverity tool and checked the errors are not reported.
Run the STM32H743ZI demo project without errors.

Checklist:
----------




By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
